modules/client/src/internal/graphql-queries/dao.ts

```diff
@@ -47,44 +47,3 @@ export const QueryDaos = gql`
     }
   }
 `;
-
-export const QueryDaoTransfersByAddress = gql`
```

Comment:
Why is this deleted? Was it leftover code?
(Just wanting to make sure this removal is not accidental)

Reply:
@Rekard0 asked me to remove it on the next PR is no longer used